web/styles/portico/integrations.css

```diff
@@ -120,9 +120,13 @@ $category-text: hsl(219deg 23% 33%);
                     border-radius: 40px;
                     border: 1px solid $light-blue-border;
                     display: block;
+                    color: hsl(0deg 0% 33%);
+                    margin-bottom: 10px;
+                    transition: border linear 0.2s, box-shadow linear 0.2s;
 
                     &:focus {
                         border: 1px solid $border-green;
+                        outline: 0;
                     }
```

Comment:
We should probably replace the `& input` in this file with a CSS class on the one input element it applies to -- as part of generally getting rid of CSS reset logic. (I don't care as much about the devtools having this problem).